### ISSUE TITLE
WCAG 4.1.2: Ensures ARIA attributes are allowed for an element's role (#aswift_1_host)

### ISSUE THREAD

**Issue by ghanmx** [MRGruas/MRGruas.github.io#6]
#### Issue
`Ensures ARIA attributes are allowed for an element's role` ([aria-allowed-attr](https://accessibilityinsights.io/info-examples/web/aria-allowed-attr))
#### Target application
[| Gruas de plataforma | M.R. Grúas | +52 8110775852 | Servicios de Asistencia Vial](https://mrgruas.github.io/)
#### Element path
#aswift_1_host
#### Snippet
`<div id="aswift_1_host" style="border: none; height: 280px; width: 1017px; margin: 0px; padding: 0px; position: relative; visibility: visible; background-color: transparent; display: inline-block; overflow: visible;" tabindex="0" title="Advertisement" a...`
#### How to fix
    Fix all of the following:
      aria-label attribute cannot be used on a div with no valid role attribute.
#### Environment
Chrome version 108.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.35.0 (axe-core 4.4.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
